bin/export-record-barcodes

```diff
@@ -60,7 +60,7 @@ def main():
         ScanProject(23089, "en", "irb-kiosk"),
     ]
 
-    with ThreadPoolExecutor(15) as pool:
+    with ThreadPoolExecutor(8) as pool:
```

Comment:
This is a good hunch, I'm interested to see if it helps! If we are still seeing timeouts after this deploys to prod, do you think it would be worth testing it with even fewer connection threads?

Reply:
Yep, I do. At that point, it might also be worth contacting ITHS to see if there's an explicit limit they've configured on their end.

I'm hoping that cutting the simultaneous connections by almost ½ will be enough, though. It'd be nice if we could reproduce locally so there wasn't a deploy/test cycle every time, but I couldn't make it happen in dev.